lib/measured/measurable.rb

```diff
@@ -66,4 +68,20 @@ def name
     end
 
   end
+
+  private
+
+  def value_string
+    @value_string ||= begin
+      str = case value
+      when Rational
+        value.denominator == 1 ? value.numerator.to_s : value.to_f.to_s
+      when BigDecimal
+        value.to_s("F")
+      else
+        value.to_f.to_s
+      end
+      str.gsub(/\.0*\Z/, "")
```

Comment:
Hmm, good question. I'm on the fence, because I'd like to represent the value with the right precision. For example, `BigDecimal("1.0000000001")` is exactly `1.0000000001`, but by doing this we'd show a value of `1`. For floats themselves it would make sense given they can't always represent certain values.